weaviate/pyproject.toml

```
@@ -0,0 +1,62 @@
+[build-system]
+requires = [
+    "hatchling>=0.13.0",
+    "setuptools>=66; python_version > '3.0'",
+    "setuptools; python_version < '3.0'",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "datadog-weaviate"
+description = "The Weaviate check"
+readme = "README.md"
+license = {text = "BSD-3-Clause"}
+requires-python = ">=3.8"
+keywords = [
+    "datadog",
+    "datadog agent",
+    "datadog check",
+    "weaviate",
+]
+authors = [
+    { name = "Datadog", email = "packages@datadoghq.com" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: BSD License",
+    "Private :: Do Not Upload",
+    "Programming Language :: Python :: 3.8",
+    "Topic :: System :: Monitoring",
+]
+dependencies = [
+    "datadog-checks-base>=25.1.0",
```

```suggestion
    "datadog-checks-base>=32.4.0",
```

You need to bump it since we added some openmetrics related stuff in newer versions